packages/vite/src/node/plugins/css.ts

```diff
@@ -1503,11 +1528,14 @@ const scss: SassStylePreprocessor = async (
   resolvers
 ) => {
   const render = loadPreprocessor(PreprocessLang.sass, root).render
+  // NOTE: `sass` always runs it's own importer first, and only falls back to
+  // the `importer` option when it can't resolve a path
```

Comment:
Let me confirm that my understanding is correct just to be sure. Here's what I understand:

- sass's own importer only resolves relative path. relative paths are correctly resolved by sass's own importer even if `window`/`location` exists.
- here Vite's custom importer handles non-relative path and resolves to an absolute path. absolute paths will be broken by sass so we need to load the content on our side.

Reply:
Yup that's exactly what happens right now 👍